oneview/resource_storage_volume_template.go

```diff
@@ -1378,7 +1378,7 @@ func resourceStorageVolumeTemplateCreate(d *schema.ResourceData, meta interface{
 			}
 			if item["enum"] != nil {
 				rawEnums := item["enum"].(*schema.Set).List()
-				enums := make([]string, len(rawEnums))
```

Comment:
why len(rawEnums)) is hard-coded to 0, hope this is not specific to one example?

Reply:
We are creating a  slice of 0 lenght and  appending to it. So having zero length slice we are starting with empty slice. Having len(rawEnums) adds blank entries in front of the appended items. This is because of difference between nil and empty slice